docs/getting-started/tests/.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+--add-opens java.base/java.lang=ALL-UNNAMED
```

Comment:
add comment why it's added.

Reply:
I don't think I can add a comment to this file. Isn't the commit message enough? If not, I'll remove it.